src/test/run-make/llvm-module-pass/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../tools.mk
+
+all:
+	$(CXX) $(LLVM_CXXFLAGS) -c llvm-pass.so.cc -o $(TMPDIR)/llvm-pass.o
+	$(AR) rcs $(TMPDIR)/libllvm-pass.a $(TMPDIR)/llvm-pass.o
+	$(RUSTC) plugin.rs --crate-name some_plugin --crate-type dylib \
+	                   -C prefer-dynamic --emit=dep-info,link \
+	                   -L native=$(TMPDIR) -l static=llvm-pass
```

Comment:
Can the flags like `-l`, `--crate-name`, and `--crate-type` be moved into the crate itself? I think you can also drop the `-L` here as it's already passed as part of `$(RUSTC)`. `--emit` can also be dropped as the dep-info isn't used